content/cloud/labels.md

```diff
@@ -0,0 +1,65 @@
+title = "Labels"
+template = "cloud_main"
+date = "2023-10-30T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/labels.md"
+
+---
+- [Labels Overview](#labels-overview)
+- [Benefits of labels](#benefits-of-labels)
+- [How To Create A Label](#how-to-create-a-label)
+- [How to Delete A Label](#how-to-delete-a-label)
```

Comment:
The only confusing part about presenting all this as labels is that `spin cloud` refers to links not labels. e.g., `spin cloud link sqlite` and `spin cloud unlink sqlite`